src/lib/menu/menu-trigger.ts

```diff
@@ -218,7 +236,11 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
       this._resetMenu();
       this._overlayRef.detach();
       this._closeSubscription.unsubscribe();
-      this.menu.close.emit();
+      if (this.menu.close) {
```

Comment:
We shouldn't need this null check either.

Reply:
See above comment.